### DISCUSSION TITLE
How to convert Cigar to its string representation?

### DISCUSSION THREAD

**Discussion by kaizhang** [zaeleus/noodles#310] [Q&A] [answered]

**Comment by zaeleus** [comment, marked as answer]
noodles tends to separate data from its at-rest form. You can use the CIGAR field writer to serialize operations into a SAM record CIGAR string. See the example at [`noodles::sam::io::writer::record::write_cigar`](https://docs.rs/noodles/0.85.0/noodles/sam/io/writer/record/fn.write_cigar.html).
